upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -209,16 +209,78 @@
+                <select name="smaily_for_opencart_rss_category" id="rss-category" class="smaily-rss-options">
+                    <option value="">All Products</option>
+                    <?php
+                    foreach ($rss_categories as $category) {
+                      $selected = (int) $category['category_id'] === $rss_category ? 'selected' : ''; ?>
+                      <option value="<?php echo $category['category_id']; ?>"<?php echo $selected; ?>><?php echo $category['name']; ?></option>
+                    <?php } ?>
+                </select>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <?php echo $rss_sort_by_title; ?>
+              </td>
+              <td>
+                <select name="smaily_for_opencart_rss_sort_by" id="rss-sort-by" class="smaily-rss-options">
+                   <?php
+                    // Add options for select.
+                    foreach ($sort_options as $sort_code => $sort_name) {
+                      $selected = $rss_sort_by === $sort_code ? 'selected' : ''; ?>
+                      <option value="<?php echo $sort_code; ?>"<?php echo $selected; ?>><?php echo $sort_name; ?></option>
+                    <?php } ?>
+                </select>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <?php echo $rss_sort_order_title; ?>
+              </td>
+              <td>
+                <select name="smaily_for_opencart_rss_sort_order" id="rss-sort-order" class="smaily-rss-options">
+                    <?php if ($rss_sort_order == 'ASC') { ?>
+                    <option value="ASC" selected="selected"><?php echo $text_ascending; ?></option>
+                    <option value="DESC"><?php echo $text_descending; ?></option>
+                    <?php } else { ?>
+                    <option value="ASC"><?php echo $text_ascending; ?></option>
+                    <option value="DESC" selected="selected"><?php echo $text_descending; ?></option>
+                    <?php } ?>
+                  </select>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <?php echo $rss_limit_title; ?>
+              </td>
+              <td>
+                <input type="number"
+                    name="smaily_for_opencart_rss_limit"
+                    min="1"
+                    max="250"
+                    id="rss-limit"
+                    value="<?php echo $rss_limit; ?>"
+                    class="smaily-rss-options" />
+                <span class="input-group-addon"><?php echo $rss_limit_products; ?></span>
+              </td>
+            </tr>
             <tr>
               <td>
                 <?php echo $rss_feed_title; ?>
               </td>
               <td>
-                <p><strong><?php echo $rss_feed_url; ?></strong></p>
+                <p><strong id="smaily-rss-feed-url" name="rss_feed_url"><?php echo $smaily_rss_url; ?></strong></p>
```

Comment:
HTML's `<strong>` tag does not have a `name` attribute. What is its purpose here?

Reply:
No purpose. Removed